.github/workflows/inference_test.yml

```diff
@@ -0,0 +1,70 @@
+name: Test Store and Inference
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'sllm/serve/**'
+      - 'sllm/cli/**'
+      - 'tests/inference_test/**'
+      - 'entrypoint.sh'
+
+jobs:
+  inference_store_tests:
+    runs-on: self-hosted
+    if: contains(github.event.pull_request.labels.*.name, 'ready-for-gpu-testing')
+    container:
+      image: pytorch/pytorch:2.3.0-cuda12.1-cudnn8-devel
+      options: --gpus all
+    defaults:
+      run:
+          shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Dependencies
+        run: |
+          apt-get update
+          apt-get install -y patch git npm wget
+
+      - name: Import Python libraries
+        run: |
+          pip install serverless-llm
+          pip install serverless-llm-store
+          pip install ray vllm
```

Comment:
I would expect the two commands above to install dependencies like ray and vllm, wouldn’t they? If they don’t, that seems like a bug.